src/app/organisms/noti-list/NotiList.tsx

```diff
@@ -154,6 +171,15 @@ function NotiList({
           );
         })}
       </div>
+      <br></br>
+      <div className="room-search__more">
+        {nextToken != null && !isLoading &&
```

Comment:
do we need to expose nextToken at all? I think we can probably expose this through `isLoading` alone

Reply:
oh good idea

Reply:
also consider using the ternary operator for this construct:

```ts
{isLoading ? <Spinner /> : <Button>Load more</Button>}
```